R/carehomes.R

```diff
@@ -726,16 +732,19 @@ carehomes_compare <- function(state, observed, pars) {
 
   ## calculate test positive probabilities for the various test data streams
   ## Pillar 2
-  pillar2_negs <- pars$p_NC * (pars$N_tot_all - model_sympt_cases)
+  time <- state["time", ]
+  p_NC <- ifelse(grepl("^S", weekdays(sircovid_date_as_date(time))),
```

Comment:
I'd recommend using the weird modulo arithmetic approach I posted before, as we'll need that for the C++ version

Reply:
Have switched to the modulo arithmetic approach, agree it's better if consistent between the two